backend.native/tests/codegen/basics/const_infinity.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package codegen.basics.const_infinity
+
+import kotlin.test.*
+
+const val fpInfConst = 1.0F / 0.0F
+val fpInfVal = 1.0F / 0.0F
+
+@Test
+fun runTest() {
+    if (fpInfConst!=Float.POSITIVE_INFINITY) throw Error()
```

Comment:
There is a special [function](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/assert-equals.html) for that. It is used in other tests.

Also it is useful to mention an origin issue in a comment.